core/templates/dev/head/dashboard/Dashboard.js

```diff
@@ -73,6 +88,26 @@ oppia.controller('Dashboard', [
       $window.location = '/create/' + explorationId;
     };
 
+    $scope.openExploration = function(status, explorationId) {
+      if (status === 'private') {
+        $scope.showExplorationEditor(explorationId);
+      } else {
+        DashboardBackendApiService.fetchExplorationStats(explorationId).then(
+          function(response) {
+            $scope.explorationStats[explorationId] = response.data;
+          }, function(errorResponse) {
+            if (FATAL_ERROR_CODES.indexOf(errorResponse.status) !== -1) {
```

Comment:
What happens if this isn't true? Perhaps we should add another warning in that case, instead of swallowing the error.